live-examples/css-examples/lists-and-counters/list-style.html

```diff
@@ -0,0 +1,58 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="list-style">
+    <div class="example-choice" initial-choice="true">
+        <pre><code id="example_one" class="language-css">list-style: square;</code></pre>
```

Comment:
Looks like the `id="example_one"` boilerplate was missed.

Reply:
Well, in reality it was not missed. The boilerplate update was introduced after this PR ;) and I was not sure if the team was going to request updates to all PRs, either way I will be cleaning that up and submitting an update.

Reply:
Yeah, totally. Awesome!